examples/parametric/plot_sed.py

```diff
@@ -51,12 +51,15 @@
 
     # create a galaxy object
     galaxy = Galaxy(stars)
-
+    
     # generate pure stellar spectra alone
     galaxy.stars.get_spectra_incident(grid)
     print("Pure stellar spectra")
+
     galaxy.plot_spectra(show=True, combined_spectra=False, stellar_spectra=True)
 
+    galaxy.plot_spectra(show=True)
```

Comment:
Is there a reason why this plot method is now called twice?

Reply:
Just a typo on my part. Thanks for pointing it out, should be fixed with the next PR.